index.js

```diff
@@ -84,22 +81,25 @@ const defaultOptions = {
   colorize: isColorSupported,
   colorizeObjects: true,
   crlf: false,
+  customColors: null,
```

Comment:
Same as with the doc block. Properties were reorganized to be in alphabetical order.